libnavigation-ui/src/main/java/com/mapbox/navigation/ui/legacy/SdkVersionChecker.kt

```diff
@@ -2,9 +2,12 @@ package com.mapbox.navigation.ui.legacy
 
 class SdkVersionChecker(private val currentSdkVersion: Int) {
 
-    fun isGreaterThan(sdkCode: Int): Boolean =
-        currentSdkVersion > sdkCode
-
```

Comment:
For documentation purposes noting that this breaks SemVer cc @langsmith

Reply:
Added it to the Migration Docs for UI SDK